Widescreen patch
Twin Castle, Main Hall (Daytime)
Knight
Stop hiding entrance knight based on y-offset.
--- original
+++ modified
@@ -16,7 +16,6 @@
 LABEL_1:
 yield
 call 257, stor[24], stor[25]
-jmp_cmp <, 0x5, stor[25], inl[:LABEL_4]
 jmp_cmp >, 0xfffffff3, stor[24], inl[:LABEL_3]
 jmp inl[:LABEL_2]
 LABEL_2:
